HIRS_Provisioner/scripts/install/hirs-provisioner.sh

```diff
@@ -9,6 +9,7 @@ CERTS_DIR="/etc/hirs/provisioner/certs/"
 HIRS_SITE_CONFIG="/etc/hirs/hirs-site.config"
 HIRS_PROVISIONER_CONFIG="/etc/hirs/provisioner/hirs-provisioner-config.sh"
 HIRS_PROVISIONER_SCRIPT="/usr/share/hirs/provisioner/bin/HIRS_Provisioner"
+HIRS_PROVISIONER_2_0_SCRIPT="/usr/local/bin/hirs-provisioner-tpm2"
```

Comment:
What was the impetus for adding all of the TPM 2.0 stuff to the older provisioner's script?

Reply:
It looks like this script is still installed on the TPM 2.0 client. I think it's used to standardize the provisioning interface.